src/compiler/types.ts

```diff
@@ -5461,7 +5461,7 @@ export const enum NodeBuilderFlags {
     AllowEmptyIndexInfoType                 = 1 << 21,
     // Errors (cont.)
     AllowNodeModulesRelativePaths           = 1 << 26,
-    
+

```

Comment:
Nothing you did I don't think but this and Dimitri's PR both had this weird diff; makes me wonder if the formatter is broken?

Reply:
I was surprised these spaces were here before—maybe the formatter improved?

Reply:
Maybe, but we enforce that all files are formatted in CI, so it shouldn't have been possible to have them at all...